src/libs/interfaces/generator/parser.cpp

```diff
@@ -588,6 +591,13 @@ InterfaceParser::parse()
 				throw InterfaceGeneratorInvalidContentException("no name for message");
 			}
 			msg_name = attr->get_value();
+#ifdef CHECK_MESSAGE_TYPE_SIZE
+			if (msg_name.length() > INTERFACE_MESSAGE_TYPE_SIZE_) {
+				throw InterfaceGeneratorInvalidContentException(
+				  "Interface message name too long, max length is %u",
+				  INTERFACE_MESSAGE_TYPE_SIZE_ - 1 - 8);
```

Comment:
Why do you check `msg_name.length() > INTERFACE_MESSAGE_TYPE_SIZE_` but then print as max length `INTERFACE_MESSAGE_TYPE_SIZE_ - 1 - 8`? Shouldn't those numbers be the same?

Reply:
You're completely right. Probably I did this wrong as afterwards the message names get modified to include to suffix "Message", which they do not yet here. Anyway, this also helped to see that the word "Message" consists of only 7 characters. (`wc -m` also counts the trailing new line.)